lib/bind/vim_shortcut.ahk

```diff
@@ -4,7 +4,7 @@
   Vim.Setting.ShowGui()
 Return
 
-; Check Mode
+; Check Mode XXX broken - what is the point of this? State.checkmode does not exist.
```

Comment:
Please revert the change of the function name.

Reply:
My mistake, thank you. I'd like to convince you to keep the new name, because as it stands, "CheckMode" is not very informative as a name. "ShowModeInfo" is much clearer to me, because "checking" usually means verifying, rather than allowing the user to check.

Reply:
This was named as `CheckMode`, because, for me at the first, it is not used normally and used only for the debugging (checking mode).

Now, it may be that ShowModeInfo seems suitable for you,
but there is a risk like what you did here.
It also make difficult to follow updates.
You should not change the function name casually if there is no critical reason.

For me, `CheckMode` is still fine and there is a no reason to change the name.

Reply:
Understood. I was changing the name because it make the code more readable, and because `CheckMode` might be confused with `ValidateMode`. I'll revert it.